glossary.html

```diff
@@ -52,28 +57,19 @@ <h1>Glossary</h1>
         <blockquote
           cite="https://www.quotetab.com/quote/by-bill-laswell/computers-and-electronic-music-are-not-the-opposite-of-the-warm-human-music-its?source=electronic-music"
         >
-          <q
-            >Computers and electronic music are not the opposite of the warm
-            human music. It's exactly the same.</q
-          >
+          Computers and electronic music are not the opposite of the warm
+            human music. It's exactly the same.
```

Comment:
No need for a `<q>` element inside a `<blockquote>`.  `<q>` is the inline version of `<blockquote>` which is a block-level element